android/KMAPro/kMAPro/src/main/java/com/keyman/android/KmpInstallMode.java

```diff
@@ -0,0 +1,7 @@
+package com.keyman.android;
```

Comment:
I never understood why SystemKeyboard.java was the only thing namespaced in com.keyman.android while everything else went into com.tavultesoft.kmapro

Reply:
I would like to move as much as possible to com.keyman.android. I think the name of the package can't change at this point... but all the internals can.

Reply:
Added #5263 for this.